english/light-fire/succumb/game-over.md

```diff
@@ -8,3 +8,5 @@ help but think about everything you're leaving behind.
 [Fight the sensation.  Claw back to life.](../../marshmallow.md)
 
 [Time to find out what heaven is like](../../marshmallow.md)
+
+[Pray for help](/get-help/help.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.